### DISCUSSION TITLE
Ignoring unused schemas

### DISCUSSION THREAD

**Discussion by nilna-iouring** [iouring-engineering/openapi-merge#15] [Ideas]
While merging with multiple open API specifications, lot of unused schemas also merged inside components, thus creates lot of additional model when generating code from merged Open API specification, can we have provision to include only used schemas in components.

**Comment by revanth-iouring** [comment]
@nilna-iouring Thanks for bringing this, we have implemented something to exclude unused schemas from the final merged Open API specification file and released new version [![npm](https://img.shields.io/npm/v/@iouring-engineering/openapi-merge-cli?label=@iouring-engineering/openapi-merge-cli&logo=npm)](https://www.npmjs.com/package/@iouring-engineering/openapi-merge-cli) , please checkout and try, always feel free to raise issues and feature requests.

**Reply by nilna-iouring** [reply]
@revanth-iouring ok,Thank you
